arch/arm/core/aarch32/Kconfig

```diff
@@ -126,6 +129,14 @@ config COMPILER_ISA_THUMB2
 config NUM_IRQS
 	int
 
+config SUPPORT_SHARED_IRQ
```

Comment:
this is (again) not something AArch32 specific. This should be moved to somewhere accessible by all the architectures.

Reply:
Removed